### PR TITLE
Mangata solo develop

### PR DESCRIPTION
**Pull request by iStrike7**

### CODE REVIEW

File: rollup/runtime/integration-test/src/xyk.rs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	});
+}
+
+#[test]
+fn create_pool_disabled_meta_disabled() {
+	test_env(Some(XykMetadata { operations_disabled: true })).execute_with(|| {
+		assert_err!(create_pool(), pallet_xyk::Error::<Runtime>::FunctionNotAvailableForThisToken);
+	});
+}
+
```
Comment (mateuszaaa) [inline rollup/runtime/integration-test/src/xyk.rs:57]: nice! are these new tests or they were moved from somewhere

File: rollup/node/src/command.rs [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+		"support.anonymous.an".into()
+	}
+
+	fn copyright_start_year() -> i32 {
+		2017
+	}
+
+	fn load_spec(&self, id: &str) -> Result<Box<dyn sc_service::ChainSpec>, String> {
+		Ok(match id {
+			"" | "rollup-local" => Box::new(chain_spec::rollup_local_config()),
```
Comment (mateuszaaa) [inline rollup/node/src/command.rs:47]: could you make sure that this config is also available
https://github.com/mangata-finance/mangata-node/pull/689

**Review by mateuszaaa — CHANGES_REQUESTED** [review]

**Review by majusko — DISMISSED** [review]

File: rollup/runtime/integration-test/src/xyk.rs (hunk 1 above)
Comment (iStrike7) [reply rollup/runtime/integration-test/src/xyk.rs:57]: These integration tests were copied from our current runtime integration tests

File: rollup/node/src/command.rs (hunk 2 above)
Comment (iStrike7) [reply rollup/node/src/command.rs:47]: This PR/branch was created from current develop so it has no rolldown stuff and hence no concept of sequencers. When we switch over to "main-ing" the solochain, I can port the rolldown stuff to it.

**Comment by mateuszaaa** [comment]
already delivered

**Pull request closed without merging**